src/pages/projects/actions/DeleteProjectBtn.tsx

```diff
@@ -41,26 +43,24 @@ const DeleteProjectBtn: FC<Props> = ({ project }) => {
 
   return (
     <ConfirmationButton
+      onHoverText="Delete project"
```

Comment:
In case the button is disabled, this should give the reason:
```
project.name === "default" ? "The default project can't be deleted" : (!isProjectEmpty(project) ? "Non empty projects can't be deleted" : "Delete project")
```

Reply:
Done, but the same should be done also for profiles and instances. I'll create a ticket for that.